omnibox.example.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - hosts: omnibox
-  user: root
+  user: ubuntu
+  sudo: true
```

Comment:
As I mentioned on Slack, Ubuntu disables the root user. Since Ubuntu is the supported distro there should either be instructions on how to enable root in the README or we should use another user (`ubuntu` is the default user on the Amazon Ubuntu AMIs).